plugins/copr.py

```diff
@@ -254,6 +269,15 @@ def _remove_repo(cls, repo_filename):
             raise dnf.exceptions.Error(str(e))
 
     @classmethod
+    def _disable_repo(cls, copr_username, copr_projectname):
+        exit_code = call(["dnf", "config-manager", "--set-disabled",
```

Comment:
Well, it does solve the use case of disabling a repository permanently using API. ...unless you want to force every plugin, which wants to disable a repository, to call something like `base.get_command('config-manager').run('--set-disabled', repo)`... which is... hm, do what you want...